src/VirtualClient/VirtualClient.Contracts/VirtualClientComponent.cs

```diff
@@ -179,7 +179,18 @@ protected VirtualClientComponent(IServiceCollection dependencies, IDictionary<st
         /// True if VC should exit/crash on first/any error(s) regardless of 
         /// their severity. Default = false.
         /// </summary>
-        public bool FailFast { get; set; }
+        public bool FailFast 
```

Comment:
In current implementation, It will conflict when both are provided. 

By default, the Failfast would not be provided for any Action/VCComponent in the profile, they will be present only in case when explicitly added in Profile. Thus, by default, the commandline argument --failfast will be picked up. But if the user decides to have specific FailFast option for one Action, they can control with this parameter, irrespective of VC CommandLine --failfast. This should give more component-wise control to users, who decide to add this parameter.

please let me know if the intent sounds good, or we should have something different.

Reply:
Do you have a use case for this or did someone requested it? If it's not a requirement let's not add it now

Reply:
I had a discussion with Bryan a couple of weeks back on it, and he had suggested to add this feature. So that we can have "Action-wise" control over the FailFast feature.